modules/thermal_hydraulics/doc/content/modules/thermal_hydraulics/tutorials/single_phase_flow/step04.md

```diff
@@ -16,14 +16,23 @@ We add two pipes for the bottom section of the primary loop with a pump in the m
 A pump is a junction-like component that connects to two flow channels corresponding to its inlet and outlet.
 
 !listing thermal_hydraulics/tutorials/single_phase_flow/04_loop.i
-         start=jct5
-         end=jct6
+         start=jct7
+         end=jct8
          link=False
          max-height=None
 
 The pump component needs 2 more parameters to be specified: reference area `A_ref`, and `head`,
 which is the pump head.
 
+
+Because the system is now closed, we add a simple pressurizer to maintain the system pressure as the fluid heats up. We split the pipe at the top to create a T-junction with a pipe connected to the pressurizer using a [VolumeJunction1Phase](source/components/VolumeJunction1Phase.md). The pressurizer is modeled by prescribing the stagnation pressure with an [InletStagnationPressureTemperature1Phase](source/components/InletStagnationPressureTemperature1Phase.md) component.
```

Comment:
```suggestion
Because the system is now closed, we add a simple pressurizer to maintain the system pressure as the fluid heats up. We split the pipe at the top to create a T-junction with a pipe connected to the pressurizer using a [VolumeJunction1Phase.md]. The pressurizer is modeled by prescribing the stagnation pressure with an [InletStagnationPressureTemperature1Phase.md] component.
```